core/src/main/scala/com/lightbend/paradox/markdown/Frontin.scala

```diff
@@ -47,7 +47,7 @@ object Frontin {
       for (s <- str) {
         val q = new java.util.Properties
         q.load(new StringReader(s))
-        p.putAll(q)
+        (p: java.util.Hashtable[AnyRef, AnyRef]).putAll(q) // https://github.com/scala/bug/issues/10418
```

Comment:
Since this function returns a `Map[String, String]`, perhaps we could replace this whole thing with
```
  case Some(s) => 
    val p = new java.util.Properties
    p.load(new StringReader(s)
    p.asScala.toMap
```
?

Reply:
done